packages/components/src/Banner/Banner.mdx

```diff
@@ -33,28 +33,80 @@ import { Banner } from "@jobber/components/Banner";
 
 ## Notice message
 
+**Use notice banners to provide information about new functionality, background
+operations, or general non-blocking advice.**
+
+Notices should get to the point and inform the user about why the functionality
+or change is important if context is not provided elsewhere.
+
 <Playground>
   <Banner type="notice">
-    Getting angry would result in a great amount of smashing
+    Your import is in progress. Feel free to leave this page while it continues.
   </Banner>
 </Playground>
 
 ## Success message
 
+**Use [toasts](toast) as the default for success messages. Success banners are
+appropriate when success feedback had a call to action or is delayed.**
+
 <Playground>
-  <Banner type="success">You've successfully Hulked out</Banner>
+  <Banner
+    type="success"
+    primaryAction={{
+      label: "View clients",
+      onClick: () => alert("🎉 Woo hoo")
+    }}
+    dismissible={false}
+  >
+    Your client import is complete
+  </Banner>
 </Playground>
 
 ## Warning message
 
+**Use warning banners when something requires when an action could have unseen
+consequences or additional action has to be taken.**
+
+Warning messages should be one to two short sentences which clearly describe the
+possibly unknown impact a user’s changes or actions will have. They should not
+begin with "Warning:" and should not be used as a way to block or redirect a
+workflow.
+
 <Playground>
-  <Banner type="warning">Smashing would result in a log of rubble</Banner>
+  <Banner type="warning">
+    Editing this schedule will replace all incomplete visits with the updated
+    information for this job
+  </Banner>
 </Playground>
 
 ## Error message
 
+**Use error banners when something requires immediate attention and blocks the
+user.**
+
+Errors should explain what happened, and summarize how many issues need to be
+addressed in a concise manner. Detailed information about each error is the role
+of the [input validation](input-validation) component.
```

Comment:
I guess there aren't a ton of specifics in here, but it might be helpful to link out to the Product Vocabulary as a"for guidance on writing error messages, check this out" resource

Reply:
I wonder if it's worth building the error-list formatting into the component itself... this is alright for now but it creates some awkward spacing since the `ul` has a margin-bottom assigned to it, that we could resolve by adding something like `bannerContent > ul { margin-bottom: 0 }`, but that still requires end users to write <ul><li></li></ul> HTML for every instance of their error banners. 

TLDR _this is fine_ but I think we can improve this at the component level